src/libs/EmojiUtils.js

```diff
@@ -278,24 +278,35 @@ function extractEmojis(text) {
     }
 
     const emojis = [];
-
-    // Text can contain similar emojis as well as their skin tone variants. Create a Set to remove duplicate emojis from the search.
-    const foundEmojiCodes = new Set();
-
     for (let i = 0; i < parsedEmojis.length; i++) {
         const character = parsedEmojis[i];
         const emoji = Emojis.emojiCodeTableWithSkinTones[character];
-
-        // Add the parsed emoji to the final emojis if not already present.
-        if (emoji && !foundEmojiCodes.has(emoji.code)) {
-            foundEmojiCodes.add(emoji.code);
+        if (emoji) {
             emojis.push(emoji);
         }
     }
 
     return emojis;
 }
 
+/**
+ * Take the current emojis and the former emojis and return the emojis that were added, if we add an already existing emoji, we also return it
```

Comment:
> if we add an already existing emoji, we also return it

I didn't quite understand this part of the comment. Can you elaborate what do you mean here?

Reply:
I mean that if we had 😏 before and now we have 😏😏😏 we need to return two 😏 even if 😏 was present in the former list